src/Manager.cpp

```diff
@@ -338,7 +338,7 @@ Manager::createDevice(const std::vector<uint32_t>& familyQueueIndices,
 
     KP_LOG_INFO("Using physical device index {} found {}",
                 physicalDeviceIndex,
-                physicalDeviceProperties.deviceName);
+                physicalDeviceProperties.deviceName.data());
```

Comment:
Why was this change required?

Reply:
It seems that formatter of `std::array<char,N>` is not included, so I take the address instead.
I know strings in std::array may not end with \0, if you think it's kind of unsafe, I can revert this change and include the corrrsponding fmt headers instead.

Reply:
The type of `deviceName` is `vk::ArrayWrapper1D<char, 256>` but not `std::array<char,N>`, fmt v10 doesn't think it is formattable, so a static assertion fails. However, previous versions of fmtlib can pass compilation, it might be caused by a change in the formatting policy.

If we don't take the address of `deviceName`, we will have to implement a formatter manually, and this will tell fmtlib how to format `vk::ArrayWrapper1D<char,N>`

Reply:
Makes sense